### PR TITLE
Add helm chart search extension

### PR DESCRIPTION
**Pull request by eunchurn**
## Description

Search in Helm Chart ([artifacthub.io](https://artifacthub.io/))

## Screencast

screenshot

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/helm-chart/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      title={searchResult.name}
+      subtitle={searchResult.description}
+      accessories={[{ text: searchResult.username }]}
```
Comment (pernielsentikaer) [inline extensions/helm-chart/src/index.tsx:50]:
```suggestion
      accessories={[{ text: searchResult.username, tooltip: searchResult.username }]}
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- The extension icon doesn't work well with dark background, you can specify both light and dark icon if  you want
> A reference to an icon file in the assets folder. Use png format with a size of 512 x 512 pixels. To support light and dark theme, add two icons, one with @dark as suffix, e.g. icon.png and icon@dark.png.

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by eunchurn** [comment]
@pernielsentikaer Thank you so much for the excellent Raycast project. My productivity has increased by over 200%, and I consistently find it very useful. The developer experience is so great that I am motivated to continue creating extensions. I truly appreciate it.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**